### PR TITLE
Clarify Rust developer scenario

### PR DESCRIPTION
**Pull request by riverar**
Rust for Windows supports GDI, DirectX, DirectDraw, various levels of XAML, etc. so the current blurb is wrong.

This change fixes that plus promotes the Rust bullet to the same level as C++ and Web.

Cherry picked out of #3793.

CC: @kennykerr

### CODE REVIEW

**Comment by PRMerger12** [comment]
@riverar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/get-started/windows-developer-faq.yml [hunk 1]
```diff
@@ -23,7 +23,8 @@ sections:
...
           - **If you're targeting cross-platform:** [React Native](https://aka.ms/reactnative) and [.NET MAUI](/dotnet/maui/what-is-maui) let you build cross-platform apps. .NET MAUI is a great choice for C# developers. React Native will feel familiar to JavaScript/TypeScript developers.
           - **If you're a web developer:** You can use [WebView2](/microsoft-edge/webview2/) as an application host. This lets you use web technology to build desktop apps that can access the Windows SDK and Windows App SDK APIs. You can also use React Native for Windows.  
-          - **If you're a C++ developer:** For desktop apps, Windows App SDK / WinUI 3 are a great choice. If you're building a game, consider using DirectX. C++ developers may also enjoy using [Rust for Windows](/windows/dev-environment/rust/rust-for-windows) to build non-GUI apps.  
+          - **If you're a C++ developer:** For desktop apps, Windows App SDK / WinUI 3 are a great choice. If you're building a game, consider using DirectX.  
+          - **If you're a Rust developer:** Check out [Rust for Windows](/windows/dev-environment/rust/rust-for-windows).
```
Comment (kennykerr) [inline hub/apps/get-started/windows-developer-faq.yml]:
```suggestion
          - **If you're a Rust developer:** Check out [Rust for Windows](https://github.com/microsoft/windows-rs).
```

The docs are pretty old at this point. Probably better just to link to the source.

**Comment by PRMerger10** [comment]
@riverar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@MatchaMatch 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

**Comment by mattwojo** [comment]
@stevewhims for awareness

**Review by stevewhims — APPROVED** [review]

**Comment by ShannonLeavitt** [comment]
@stevewhims - Thanks for approving the PR. For future PRs, please note that a `#sign-off` comment is the best way to signal to the review team that a PR is ready to merge.

I've resolved a merge conflict and will merge now.

**Comment by stevewhims** [comment]
Not sure what's happened to this PR. The "Files changed" tab says "These merge commits were added into this branch cleanly.". I've never seen that before. :) Just to get this one off the books, I'll take a chance and click "Squash and merge", and see what happens.

**Comment by riverar** [comment]
@stevewhims It's because @ShannonLeavitt's commits resolved the merge conflict incorrectly and it blew away all the changes.

**Comment by stevewhims** [comment]
> @stevewhims It's because @ShannonLeavitt's commits resolved the merge conflict incorrectly and it blew away all the changes.

Oh! Are you able to recreate the PR? I'll be happy to go ahead and merge it. If you can assign just myself that might be easiest.

Many thanks, and sorry about the extra work.

-Steve

**Comment by ShannonLeavitt** [comment]
@riverar @stevewhims - Thanks for raising the issue of the incorrect resolution of the merge conflicts. I'm not exactly sure how I messed it up. My apologies for the extra work!

**Pull request merged**